nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-content-viewer/src/main/java/org/apache/nifi/web/ContentViewerController.java

```diff
@@ -150,6 +150,7 @@ protected void doGet(final HttpServletRequest request, final HttpServletResponse
         // buffer the content to support reseting in case we need to detect the content type or char encoding
         try (final BufferedInputStream bis = new BufferedInputStream(downloadableContent.getContent());) {
             final String mimeType;
+            final String normalizedMimeType;
 
             // when standalone and we don't know the type is null as we were able to directly access the content bypassing the rest endpoint,
             // when clustered and we don't know the type set to octet stream since the content was retrieved from the node's rest endpoint
```

Comment:
@mcgilman - Do you mean that this part:
`downloadableContent.getType().equals(MediaType.OCTET_STREAM.toString())`
need the same transformation applied as normalizedMimeType further down, before comparing it to anything?

If so, then yes, I see what you mean and fully agree, please feel free to make the change as you see fit.